src/components/pages/Profile.svelte

```diff
@@ -34,6 +37,8 @@
     <p>Fetching user data...</p>
   {:then userData}
     <ProfileData {userData} />
+    <Follow label="Following" users={["660743113622a1894e2b7d98"]} firstName={["testy"]} lastName={["McTesterson"]} />
+    <Followers label="Followers" users={["660743113622a1894e2b7d98"]} firstName={["testy"]} lastName={["McTesterson"]} />
```

Comment:
you can just reuse the `<Follow>` component here. It is functionally exactly the same as the `<Followers>` component you made, and so there is no need for two different components.